front-end-101/final_project_plan/entertainment.html

```diff
@@ -26,7 +26,7 @@
     </header>
     <main style="height: 10vh; display: grid; place-items: center;">
         <h2>Список любимых аниме</h2>
-        <div class="entertainment"><!--Список аниме-->
+        <div class="entertainment" id="entertainment"><!--Список аниме-->
```

Comment:
same here use either class or id. Using both does not make sense since you have similar names and purpose os the styles